db/migrate/20171024183226_create_books.rb

```diff
@@ -1,6 +1,7 @@
 class CreateBooks < ActiveRecord::Migration[5.1]
   def change
     create_table :books do |t|
+      t.rating :integer
```

Comment:
Watch out, it is dangerous to add a line to an old migration. This migration file will not get run when you call rake db:migrate since rake looks for only migration files that haven't been run already. This would then cause problems if you every try to roll back this far since the undo directions are now different from the direction that did the initial creation. You ALWAY want to create a new migration to add a column rather than adding to an old migration file.